tests/gold_tests/pluginTest/traffic_dump/traffic_dump.test.py

```diff
@@ -99,7 +101,7 @@
 
 tr.Processes.Default.StartBefore(server, ready=When.PortOpen(server.Variables.Port))
 tr.Processes.Default.StartBefore(Test.Processes.ts)
-tr.Processes.Default.Command = 'curl http://127.0.0.1:{0} -H\'Host: www.example.com\' --verbose'.format(
+tr.Processes.Default.Command = 'curl http://127.0.0.1:{0} -H\'Cookie: donotlogthis\' -H\'Host: www.example.com\' --verbose'.format(
```

Comment:
It seems whoever originally wrote this developed a compulsive aversion to using double quotes after line 98, and you're respecting that.  Whatever the cost in backslashes.

Reply:
Good point. Using double quotes makes it more readable.